WordPress/WordPressTest/Services/PostCoordinatorFailedPostsFetcherTests.swift

```diff
@@ -74,8 +74,8 @@ private extension PostCoordinator.FailedPostsFetcher {
     func getPostsToRetrySync() -> [AbstractPost] {
         var result = [AbstractPost]()
```

Comment:
Will this test be updated later on to return the result as a dictionary of posts and their actions?

Reply:
Probably not. There will be more specific tests for the actions in `PostCoordinatorUploadActionUseCaseTests`. The `FailedPostsFetcher` just tests the Core Data query for now and I think that's fine.